stack.yaml

```diff
@@ -11,56 +11,97 @@
 # resolver: lts-3.5
```

Comment:
All this dead code and obsolete comments don't seem to serve any purpose and make it hard to find what's actually live. Can it be removed?